main.cpp

```diff
@@ -62,64 +62,93 @@ send me a DM to check your pull request
 #include <string>
 struct T
 {
-    T(<#type name#> v, const char* <#variable name#>)   //1
+    int value;
+    std::string name;
+
+    T(int v, const char* charPointer)
+    {
+        value = v;
+        name = charPointer;
+    };  //1
```

Comment:
Good. you could have also used the member initializer list.
no need for the semicolon at the end.
- [ ] remove the semicolon after the `}`